### ISSUE TITLE
flexi-project: no ISNAN GNU extension provided by flang

### ISSUE THREAD

**Issue by pawosm-arm** [flang-compiler/flang#299]
As in the topic. Flang can't be used as gfortran drop-in replacement because of that.

**Comment by janusw** [comment]
But: flang seems to know about `ieee_is_nan` from `ieee_arithmetic`, which is even better, because it's standard F2003.

**Comment by pawosm-arm** [comment]
We all agree it would be best if Flang could be used as gfortran drop-in replacement, so it needs both.

**Comment by mvinay** [comment]
Is there a plan for supporting ISNAN ?

**Comment by mvinay** [comment]
@sscalpone Yes, I am able to use but I want to compile without changing the source code written for gfortran.. Are there plans for supporting it?
